extensions/google-translate/src/simple-translate.ts

```diff
@@ -129,7 +129,9 @@ export async function playTTS(text: string, langTo: string) {
         if (code !== 0) {
           console.error("Error playing audio");
         }
-        unlinkSync(tempFilePath);
+        if (existsSync(tempFilePath)) {
+          unlinkSync(tempFilePath);
+        }
```

Comment:
what exactly is happening here if we are not using `unlinkSync`? Because I'm trying to run this multiple time, and nothing critical is happening, it is just playing multiple times

Reply:
If holding cmd+t you can end up in this crash @gebeto 

![Uploading Raycast26022024-erdPgMDd.png@2x.png…]()

Reply:
yep, it makes sense